trymito.io/styles/ExcelToPython.module.css

```diff
@@ -0,0 +1,147 @@
+.main {
+    align-items: flex-start !important; 
+    position: relative;
+}
+
+.title_card {
+    padding-top: 4rem;
+}
+
+.title_card > *:not(:last-child) {
+    margin-bottom: 2rem;
+}
+
+.section {
+    align-items: flex-start !important;
+}
+
+.section + .section {
+    margin-top: 4rem;
+}
+
+.related_functions_card {
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
+}
+
+.related_functions_card * {
+    margin-right: 10px;
+}
+
+.h3 {
+    font-family: Graphik;
+    font-style: normal;
+    font-weight: 400;
+    margin-top: 2rem;
+    font-size: 1.25rem;
+}
+
+.table {
```

Comment:
Same with these. Class names the same as elements -- let's not do this to ourselves